File: app/src/main/java/ch/bailu/aat/services/tileremover/State.java
package ch.bailu.aat.services.tileremover;

public interface State {

    /** firstPixelIndex scanning **/
    void scan();

    /** chancel **/
    void stop();

    /** setItem to initial state **/
    void reset();

    /** firstPixelIndex removing **/
    void remove();

    /** rebuild list of files to remove
     * (trim settings have been changed) **/
    void rescan();

    /** remove all tiles in selected directory **/
    void removeAll();
}

File: aat-android/src/main/kotlin/ch/bailu/aat/services/tileremover/State.kt
package ch.bailu.aat.services.tileremover

interface State {
    /** firstPixelIndex scanning  */
    fun scan()

    /** chancel  */
    fun stop()

    /** setItem to initial state  */
    fun reset()

    /** firstPixelIndex removing  */
    fun remove()

    /** rebuild list of files to remove
     * (trim settings have been changed)  */
    fun rescan()

    /** remove all tiles in selected directory  */
    fun removeAll()
}
